### ISSUE TITLE
ImportOrder checking impossible with matchall group not being the last group

### ISSUE THREAD

**Issue by anilkumarmyla** [scalastyle/scalastyle#325]
Currently its impossible to do below check since `others` is a matchall group and gobbles up scala group as well
```
    <check level="warning" class="org.scalastyle.scalariform.ImportOrderChecker" enabled="true">
        <parameters>
            <parameter name="groups">java,others,scala</parameter>
            <parameter name="group.java">javax?\..+</parameter>
            <parameter name="group.others">.+</parameter>
            <parameter name="group.scala">scala\..+</parameter>
        </parameters>
    </check>
```

Input
```
import java.util.x

import com.a.b.c.x

import scala.util.x
```

Expected output - no warnings
Actual output - There should be no empty line separating imports in the same group (on scala import line).

**Comment by anilkumarmyla** [comment]
workaround to have others as ` <parameter name="group.others">^(?!scala\.).+</parameter>`
